### ISSUE TITLE
[Feature request] Ingredient price multiplier

### ISSUE THREAD

**Issue by Aberro** [ShadowTheAge/yafc#96]
Just some kind of combobox (x0.5, x1, x2, x10) or up-down numeric box which will be used as a multiplier to YAFC cost, so we can set some abundant ingredients (like water or electricity) as preferred over some scarce ones.

**Comment by ShadowTheAge** [comment]
Non-linked ingredients have no affect on calculations so changing their costs will do nothing
YAFC uses recipe costs (not ingredient costs) as objective when the solution is not single-point
The ability to customize objective function will be useful in some cases. But the only way that I see that could be done somewhat intuitively is to add some sort of "maximize usage / default / minimize usage" select

that does not however solve all the issues because how do we choose between tow "maximize usage" recipes?

**Comment by ShadowTheAge** [comment]
Also I can try to emulate objective function "minimize recipe cost" but that will require experiments
